bot/exts/evergreen/pythonfacts.py

```diff
@@ -0,0 +1,23 @@
+import random
+
+import discord
+from discord.ext import commands
+from discord.ext.commands.bot import Bot
+
+
+class PythonFacts(commands.Cog):
+    """Gives a random fun fact about Python."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @commands.command(name='pythonfact', aliases=['pyfact'])
+    async def get_python_fact(self, ctx: commands.Context) -> None:
+        """Gives a Random fun fact about Python."""
+        with open('bot/resources/evergreen/python_facts.txt') as file:
```

Comment:
Ideally we should load this file into a variable when loading the cog, rather than every time this command is ran.

Reply:
How would I do that?

Reply:
https://github.com/python-discord/sir-lancebot/blob/3ad829d98f1de30f08d26fe592b8a89701301983/bot/exts/easter/egg_decorating.py#L15-L16 is an example of this being done. As you can see they're loaded into a constant outside of the cog.

Reply:
Done